src/angular.prefix

```diff
@@ -3,4 +3,4 @@
  * (c) 2010-2016 Google, Inc. http://angularjs.org
  * License: MIT
  */
-(function(window, document, undefined) {
+(function(window) {
```

Comment:
I know what it does. :) What I'm saying is that it doesn't really buy us much; there are thousands other APIs that, if messed with, will break Angular and we don't protect against such changes. Moreover, I think it's more likely that someone will break `Object.prototype` for us than that they'll shadow `undefined`. I feel this `undefined` parameter is only providing a false sense of security.